docs/web3j-openapi.md

```diff
@@ -0,0 +1,248 @@
+      "description": "List HelloWorld method&#39;s calls"
+    }
+  ],
+  "paths": {
+    "/HelloWorldProject/contracts/helloworld": {
+      "post": {
+        "tags": [
+          "HelloWorld"
+        ],
+        "summary": "Deploys the HelloWorld contract",
+        "operationId": "deploy",
+        "responses": {
+          "default": {
+            "description": "default response",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/TransactionReceipt"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    ...
+```
+
+With a corresponding `Swagger-UI` : 
+![image](img/Web3j-OpenAPI/SwaggerUI_1.png)
+
+and the available methods:
+
+![image](img/Web3j-OpenAPI/SwaggerUI_2.png)
+
+## Use cases
+An OpenAPI project can be generated using the [Epirus-CLI](https://github.com/epirus-io/epirus-cli) as follows:
+
+```ssh
+$ epirus generate openapi -p com.helloworld -o . --abi helloworld.abi --bin helloworld.bin --name helloworld
+```
+
+Then, the generated project can be used in the following ways:
+
+**Creating a Gradle project that can be run using the application plugin:**
+```ssh
+$ cd helloworld
+$ ./gradlew run // Starts the server exposing Helloworld.sol
+...
+```
+
+**Starting the server using the ShadowJar:**
+```ssh
+$ ./gradlew shadowJar // Create an executable JAR that starts the server
+$ java -jar helloworld-server-all.jar
+...
+```
+
+**Using the CLI:**
+
+```ssh
+$ ./gradlew installDist // Create an executable to start the server
+$ ./helloworld-server
+```
+
+# Getting started with Web3j-OpenAPI
+
+## How to generate a project:
+To generate an OpenAPI project using the [Web3j-OpenAPI](https://github.com/web3j/web3j-openapi) generator, you need to have the [Epirus-CLI](https://github.com/epirus-io/epirus-cli) installed on your machine (Note - the Epirus CLI has replaced the Web3j CLI). 
+It’s easy to do (for Windows instructions head [here](https://docs.epirus.io/quickstart/#installation)):
+	
+```
+curl -L get.epirus.io | sh
+```
+
+And then, you can run the
+```
+Epirus generate openapi --help
+```
+To check the generator available options. 
+
+![image](img/Web3j-OpenAPI/Generator_help.png)
+
+In this case, we will be using the above Hello World contract. 
+To compile it, you can use solc via [SVM](https://blog.web3labs.com/a-solidity-version-manager-using-sokt). This lets you handle multiple solidity compiler versions easily:
+```
+Svm install 0.6.0
+Svm use 0.6.0
+Solc --abi --bin -o . HelloWorld.sol
+```
+
+Now, you should  see the ABI and Bin files in the directory. You  can use them to generate the OpenAPI:
+```	
+Epirus generate openapi --abi HelloWorld.abi --bin HelloWorld.bin --package-name com.tutorial --output . --name HelloWorldProject
+```
+You should be seeing logs similar to the following:
+![image](img/Web3j-OpenAPI/Generator_logs.png)
+
+## How to build
+
+You have two possibilities:
```

Comment:
You should recommend that people use one of the two approaches - you don't want them to think "which one of these approaches should I use?"